app/controller.h

```diff
@@ -0,0 +1,28 @@
+/// \file
+/// Maintainer: Felice Serena
+///
+///
+
+#pragma once
+
+#include "pipeline.h"
+
+namespace MouseTrack {
+
+class Controller {
+public:
+
+    /// Called by main after creation
+    virtual int main(int argc, char *argv[]) = 0;
```

Comment:
This might be exchanged by a object resulting from parsing the command line, don't use it for the moment